src/etc/inc/system.inc

```diff
@@ -1503,6 +1503,49 @@ function system_timezone_configure() {
 	}
 }
 
+function check_gps_speed($device) {
+	usleep(1000);
+	// Set timeout to 5s
+	$timeout=microtime(true)+5;
+	if($fp = fopen($device, 'r')) {
+		stream_set_blocking($fp, 0);
+		stream_set_timeout($fp, 5);
+		$contents = "";
+		$cnt = 0;
+		$buffersize = 256;
+		do {
+			$c = fread($fp, $buffersize - $cnt);
+
+			// Wait for data to arive
+			if($c === false || strlen($c) == 0) {
```

Comment:
Needs a space between `if` and `(`.
Each test in the `if` needs its own set of parenthesis.